test/github.test.js

```diff
@@ -5,37 +5,33 @@ const fs = require('fs');
 const github = require('@actions/github');
 const sinon = require('sinon');
 const yaml = require('yaml');
+
+// rewire's method name needs to be disabled from lint since we don't control it
+/* eslint no-underscore-dangle: ["error", { "allow": ["__set__"] }] */
+const rewire = require('rewire');
+
 const { ContextStub } = require('./stubs/context');
 const { expect } = require('chai');
 
-const {
-  get_pull_request,
-  fetch_config,
-  fetch_changed_files,
-  fetch_current_reviewers,
-  assign_reviewers,
-  clear_cache,
-} = require('../src/github');
-
 describe('github', function() {
+  const local_github = rewire('../src/github');
```

Comment:
```suggestion
  const rewired_github = rewire('../src/github');
```

I'm not in love with this variable name. "Local" makes it sound like the actual GitHub instance is somehow local and that is what we're getting an API to.

Could we consider renaming this to "rewired_github" or something that is perhaps more explicit? I personally wouldn't be entirely opposed to just calling it "github" either if that's possible.

Reply:
unfortunately, we also import the actual github action for config mocking, so I had to add a prefix (I would have much preffered just github :( ).

`rewired_github` sounds better than local though, I can go with that